docs/automated-process.md

```diff
@@ -11,6 +11,29 @@ release process. The automated processes uses GitHub Actions to automate the
     `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
     specify.
 
+## Setup for the release
+
+### Credentials
+
+- Membership to [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.
+- Become member of the [Ansible Release Management working group](https://github.com/orgs/ansible-community/teams/release-management-wg/).
+- Join [Release Management Working Group in [Ansible Forum](https://forum.ansible.com/g/release-managers) and [Matrix](#release-management:ansible.com).
```

Comment:
For the release announcements, you need to be member of more forum groups. Should these also be mentioned here?